### PR TITLE
95 instances are sometimes wrong

### PR DESCRIPTION
**Pull request by MircoNierenz**
# Description

fix instance.json's
- remove shape in name
- media, requiredData, relatedData as lists
- remove duplicate link entries
- valid coordinates for georeference:BoundingBox/origin

## Type of change

Please delete options that are not relevant.

- [ ] New type (non-breaking change which adds a type)
- [x] Change (non-breaking change or fix on an existing type)
- [ ] Breaking change (Change that would cause existing Self Descriptions not to be accepted anymore by a Federated Catalogue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any
relevant details for your test configuration

- https://shacl.org/playground/
- provider-tools\jsonLD_validator

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

### CODE REVIEW

**Comment by MircoNierenz** [comment]
add shape to name in @type to be valid with sh:closed = true for environment-model. 
I am working on the other jsonLDs now.

File: environment-model/environment-model_shacl.ttl [hunk 1]
```diff
@@ -4,9 +4,12 @@
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
 environment-model:EnvironmentModelShape a sh:NodeShape ;
-    sh:property [ sh:maxCount 1 ;
+    sh:closed false ;
+	sh:ignoredProperties ( rdf:type ) ;
```
Comment (lenasauermann) [inline environment-model/environment-model_shacl.ttl:11]: komische Einrückungen

Comment (lenasauermann) [reply environment-model/environment-model_shacl.ttl:11]: markiere ich mal nur hier, ist aber an vielen Stellen der Fall

**Review by lenasauermann — CHANGES_REQUESTED** [review]

File: environment-model/environment-model_shacl.ttl (hunk 1 above)
Comment (MircoNierenz) [reply environment-model/environment-model_shacl.ttl]: gefixt - im notepad++ war default tab eingestellt

File: georeference/georeference_shacl.ttl [hunk 2]
```diff
@@ -108,7 +115,10 @@ georeference:BoundingBoxShape a sh:NodeShape ;
             sh:path georeference:yMax ];
     sh:targetClass georeference:BoundingBox .
 
+
```
Comment (robertschubert) [inline georeference/georeference_shacl.ttl:118]: Bitte unnötige Leerzeile löschen

**Review by robertschubert — CHANGES_REQUESTED** [review]
Nur eine Kleinigkeit, siehe Kommentar.

File: georeference/georeference_shacl.ttl (hunk 2 above)
Comment (MircoNierenz) [reply georeference/georeference_shacl.ttl]: done!

**Review by robertschubert — APPROVED** [review]

**Review by lenasauermann — APPROVED** [review]

**Comment by robertschubert** [comment]
2x approved - merge

**Pull request merged**